design/login.md

```diff
@@ -51,6 +51,10 @@ reading the `WWW-Authenticate` header DC/OS returns in the 401 response:
 }
 ```
 
+If the `Head` request returns a 200 response, the cluster has no authorization. Setup will succeed and
```

Comment:
"the cluster does not require authentication"  would be much more correct than "the cluster has no authorization"